src/strategies/ocean-dao-brightid/index.ts

```diff
@@ -0,0 +1,71 @@
+import { getScoresDirect, multicall, getProvider } from '../../utils';
+
+export const author = 'trizin';
+export const version = '0.1.0';
+
+const abi = [
+  'function isVerifiedUser(address _user) external view returns (bool)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const ALLOWED_STRATEGIES = [
+    'erc20-balance-of',
+    'ocean-marketplace',
+    'sushiswap',
+    'uniswap',
+    'contract-call'
+  ];
+  if (
+    options.strategies.length > 5 ||
+    options.strategies.some((x) => !ALLOWED_STRATEGIES.includes(x.name))
+  )
+    return [];
+  const response = await multicall(
+    options.brightIdNetwork,
+    getProvider(options.brightIdNetwork),
+    abi,
+    addresses.map((address: any) => [
+      options.registry,
+      'isVerifiedUser',
+      [address]
+    ]),
+    { blockTag: 'latest' }
```

Comment:
This wont work, you need to use the snapshot block number otherwise the score may change anytime and proposals results would also change. The score should always be the same if you compute it now or next week unless the param "snapshot" is equal to "latest"

Reply:
@bonustrack Thanks for pointing that out! I've changed it to `blockTag: snapshot`.